code/artificial_intelligence/src/neural_network/lms/README.md

```diff
@@ -0,0 +1 @@
+# LMS with Sigmoid Function
```

Comment:
More information, please.

Reply:
Sure. I'm on it. Sorry, this is my first PR.